# Mini-COES "Think tanks Chilenos en tiempos turbulentos (2019-2023)"
**Proyecto: minicoesfondap-1523A0005**

## Introducción
Este informe presenta una descripción detallada de la base de datos recopilada como parte del proyecto *Mini-COES "Think tanks Chilenos en tiempos turbulentos (2019-2023)"*.
El propósito de este documento es caracterizar de manera objetiva la base de datos, describiendo sus variables, estructura, y proceso de creación.


In [1]:
import pandas as pd

file_path = "Database/01_bbdd_think_tanks.xlsx" #por favor usar el path correcto al archivo ubicado en ./Database
df = pd.read_excel(file_path)

df.head()

ID Think Tank Tipo de Think Tank Orientación Política Autor  \
0   1        LyD        Transversal              Derecha   NaN   
1   2        LyD        Transversal              Derecha   NaN   
2   3        LyD        Transversal              Derecha   NaN   
3   4        LyD        Transversal              Derecha   NaN   
4   5        LyD        Transversal              Derecha   NaN   

                                              Título Medio  \
0  TIERRAS INDÍGENAS: DISCRECIONALIDAD A LA HORA ...   NaN   
1  PROYECTO DE LEY QUE CREA UN SISTEMA MIXTO DE P...   NaN   
2  CRISIS DE SEGURIDAD EN CHILE: LA URGENCIA DE D...   NaN   
3                LEY DE INTELIGENCIA PASA A HACIENDA   NaN   
4  PRINCIPALES ALERTAS FRENTE AL INFORME DE SOSTE...   NaN   

                                              Corpus  Producto  \
0  Hace más de 30 años se promulgó en nuestro paí...  Noticias   
1  Nuestro sistema de pensiones requiere de mejor...  Noticias   
2  Chile enfrenta una crisis de seguridad que req...  Noticias   
3  El proyecto de ley que fortalece y moderniza e...  Noticias   
4  Análisis preliminar. Borrador. 22/1/2025 A. CF...  Noticias   

                                              Enlace EnlacePDF CorpusPDF  \
0  https://lyd.org/centro-de-prensa/noticias/2025...       NaN       NaN   
1  https://lyd.org/centro-de-prensa/noticias/2025...       NaN       NaN   
2  https://lyd.org/centro-de-prensa/noticias/2025...       NaN       NaN   
3  https://lyd.org/centro-de-prensa/noticias/2025...       NaN       NaN   
4  https://lyd.org/centro-de-prensa/noticias/2025...       NaN       NaN   

  FechaPublicación2     año FechaPublicacion     Año  Mes   Dia  \
0        2025-01-30  2025.0       2025-01-30  2025.0  1.0  30.0   
1        2025-01-27  2025.0       2025-01-27  2025.0  1.0  27.0   
2        2025-01-27  2025.0       2025-01-27  2025.0  1.0  27.0   
3        2025-01-23  2025.0       2025-01-23  2025.0  1.0  23.0   
4        2025-01-22  2025.0       2025-01-22  2025.0  1.0  22.0   

                                               Texto  \
0  TIERRAS INDÍGENAS: DISCRECIONALIDAD A LA HORA ...   
1  PROYECTO DE LEY QUE CREA UN SISTEMA MIXTO DE P...   
2  CRISIS DE SEGURIDAD EN CHILE: LA URGENCIA DE D...   
3  LEY DE INTELIGENCIA PASA A HACIENDA. El proyec...   
4  PRINCIPALES ALERTAS FRENTE AL INFORME DE SOSTE...   

                                         TextoLimpio  
0  tierras indigenas discrecionalidad hora asigna...  
1  proyecto ley crea sistema mixto pensiones segu...  
2  crisis seguridad urgencia dotar fuerzas polici...  
3  ley inteligencia pasa hacienda proyecto ley fo...  
4  principales alertas frente informe sostenibili...

## Descripción General de la Base de Datos

### Alcance
- **Centros de Estudio Incluidos**:
  - **Izquierda**
    - Transversales:
      - Fundación Sol
      - Nodo XXI
      - Chile 21
      - Espacio Público
      - Centro de Estudios del Desarrollo (CED)
      - Centro de Estudios Nacionales de Desarrollo Alternativo (CENDA)
      - Horizonte Ciudadano
    - De partido:
      - Rumbo Colectivo
      - Instituto Igualdad
      - Centro Democracia y Comunidad
      - Instituto de Ciencias Alejandro Lipschutz (ICAL)
      - Observatorio de Políticas Económicas (OPES)
    - Semi Difuntos:
      - Corporación de Estudios para Latinoamérica (CIEPLAN)
      - La Casa Común
      - Saberes Colectivos
  - **Derecha**
    - Transversales:
      - Libertad y Desarrollo (LyD)
      - Centro de Estudios Públicos (CEP)
      - Pivotes
      - Instituto de Estudios de la Sociedad (IES)
      - IdeaPaís
      - Instituto ResPública
      - Fundación para el Progreso (FPP)
    - De partido:
      - Fundación Jaime Guzmán
      - Horizontal
      - Ideas Republicanas
      - Instituto Libertad
    - Universidades:
      - Faro UDD
      - Signos UAndes
      - Centro Latinoamericano de Políticas Económicas y Sociales (Clapes UC)




In [9]:
df.shape

(28889, 11)

- **Número de Registros**: 28889
- **Periodo de Cobertura**: Todas las publicaciones históricas.
- **Origen de Datos**: Publicaciones reportadas en su página web.


## Estructura de la Base de Datos

La base de datos contiene las siguientes variables principales:

| **Variable**             | **Descripción**                                                                  | **Tipo de Dato**    |
|--------------------------|----------------------------------------------------------------------------------|---------------------|
| **ID**                   | Identificador único para cada registro en la base de datos.                      | Numérico (Ordinal)  |
| **Think Tank**           | Nombre del centro de estudios asociado a cada registro.                         | Categórico (Texto)  |
| **Tipo de Think Tank**   | Clasificación según su naturaleza: partido, transversal, o universidad.          | Categórico (Texto)  |
| **Orientación Política** | Clasificación ideológica del think tank (Derecha o Izquierda).                   | Categórico (Texto)  |
| **Autor**                | Nombre del autor o autores de la publicación (si está disponible).               | Texto               |
| **Título**               | Título de la publicación.                                                        | Texto               |
| **Fecha de Publicación** | Fecha en la que se realizó la publicación.                                       | Temporal (Fecha)    |
| **Medio de Comunicación**| Medio donde se publicó la intervención (si aplica).                              | Categórico (Texto)  |
| **Corpus**               | Contenido del documento publicado.                                               | Texto               |
| **Producto**             | Tipo de documento publicado: Carta al Director, Columna, Documento Académico, etc.| Categórico (Texto)  |
| **Enlace**               | Enlace al recurso digital de la publicación.                                     | Texto (URL)         |


In [10]:
df.dtypes

ID                        int64
Think Tank               object
Tipo de Think Tank       object
Orientación Política     object
Autor                    object
Título                   object
Fecha de Publicación     object
Medio de Comunicación    object
Corpus                   object
Producto                 object
Enlace                   object
dtype: object


## Cobertura
- **Periodo Cubierto**: La base de datos abarca intervenciones realizadas entre octubre de 2019 y diciembre de 2023. Los períodos clave incluyen:
  - **18 de Octubre de 2019**: Inicio del estallido social.
  - **15 de Noviembre de 2019**: Acuerdo por la Paz.
  - **18 de Marzo de 2020**: Decreto de Estado de Excepción por la pandemia de COVID-19.
  - **27 de Octubre de 2020**: Plebiscito de Entrada.
  - **16 de Mayo de 2021**: Elección de Convencionales Constituyentes.
  - **4 de Julio de 2021**: Inicio de la Convención Constitucional.
  - **18 de Octubre de 2021**: Inicio de la Discusión Temática en la Convención.
  - **15 de Febrero de 2022**: Inicio de los Plenos en la Convención.
  - **29 de Abril de 2022**: Inicio de las Comisiones Finales (Armonización, Preámbulo, Normas Transitorias).
  - **4 de Julio de 2022**: Entrega oficial del proyecto constitucional.
  - **4 de Septiembre de 2022**: Plebiscito de Salida.
  - **12 de Diciembre de 2022**: Publicación del Documento de Bases para el nuevo proceso constituyente.
  - **24 de Enero de 2023**: Lista de 24 expertos designados para el proceso.
  - **6 de Marzo de 2023**: Inicio de la Comisión Experta.
  - **7 de Mayo de 2023**: Elección de Consejeros Constitucionales.
  - **30 de Mayo de 2023**: Fin del Anteproyecto Constitucional.
  - **7 de Junio de 2023**: Inicio de sesiones del Consejo Constitucional.
  - **7 de Noviembre de 2023**: Cierre del Consejo Constitucional.
  - **17 de Diciembre de 2023**: Plebiscito de Salida del segundo proceso constituyente.



### Representatividad
- **Centros de Estudio Representados**: 
  - **Izquierda**: La base de datos incluye **19 centros de estudio** asociados a orientaciones progresistas o de izquierda:
    - **Transversales (7)**: Fundación Sol, Nodo XXI, Chile 21, Espacio Público, Centro de Estudios del Desarrollo (CED), Centro de Estudios Nacionales de Desarrollo Alternativo (CENDA), Horizonte Ciudadano.
    - **De partido (6)**: Rumbo Colectivo, Instituto Igualdad, Centro Democracia y Comunidad, Instituto de Ciencias Alejandro Lipschutz (ICAL), Observatorio de Políticas Económicas (OPES).
    - **Semi Difuntos (3)**: Corporación de Estudios para Latinoamérica (CIEPLAN), La Casa Común, Saberes Colectivos.
  - **Derecha**: La base de datos incluye **16 centros de estudio** asociados a orientaciones conservadoras o de derecha:
    - **Transversales (7)**: Libertad y Desarrollo (LyD), Centro de Estudios Públicos (CEP), Pivotes, Instituto de Estudios de la Sociedad (IES), IdeaPaís, Instituto ResPública, Fundación para el Progreso (FPP).
    - **De partido (5)**: Fundación Jaime Guzmán, Horizontal, Ideas Republicanas, Instituto Libertad.
    - **Universidades (3)**: Faro UDD, Signos UAndes, Centro Latinoamericano de Políticas Económicas y Sociales (Clapes UC).


In [11]:
import plotly.express as px

orientation_counts = df['Orientación Política'].value_counts().reset_index()
orientation_counts.columns = ['Orientación Política', 'Número de IDs']

fig = px.bar(orientation_counts, x='Orientación Política', y='Número de IDs', 
             title='Número de IDs por Orientación Política', 
             labels={'Orientación Política':'Orientación Política', 'Número de IDs':'Número de IDs'},
             color='Orientación Política', color_discrete_map={'Derecha':'red', 'Izquierda':'blue'})

fig.show()
fig = px.pie(orientation_counts, names='Orientación Política', values='Número de IDs', 
             title='Distribución de IDs por Orientación Política', 
             color='Orientación Política', color_discrete_map={'Derecha':'red', 'Izquierda':'blue'})

fig.show()

In [12]:
think_tank_counts = df['Think Tank'].value_counts().reset_index()
think_tank_counts.columns = ['Think Tank', 'Número de IDs']
think_tank_counts = think_tank_counts.sort_values(by='Número de IDs', ascending=True)

fig = px.bar(think_tank_counts, x='Número de IDs', y='Think Tank', 
             title='Número de IDs por Think Tank', 
             labels={'Think Tank':'Think Tank', 'Número de IDs':'Número de IDs'},
             orientation='h', height=800)

fig.show()


In [13]:
tipo_think_tank_counts = df['Tipo de Think Tank'].value_counts().reset_index()
tipo_think_tank_counts.columns = ['Tipo de Think Tank', 'Número de IDs']
tipo_think_tank_counts = tipo_think_tank_counts.sort_values(by='Número de IDs', ascending=False)

fig = px.bar(tipo_think_tank_counts, x='Número de IDs', y='Tipo de Think Tank', 
             title='Número de IDs por Tipo de Think Tank', 
             labels={'Tipo de Think Tank':'Tipo de Think Tank', 'Número de IDs':'Número de IDs'},
             color='Tipo de Think Tank', orientation='h')

fig.show()

In [14]:

izquierda_tipo_counts = df[df['Orientación Política'] == 'Izquierda']['Tipo de Think Tank'].value_counts().reset_index()
izquierda_tipo_counts.columns = ['Tipo de Think Tank', 'Número de IDs']
izquierda_tipo_counts = izquierda_tipo_counts.sort_values(by='Número de IDs', ascending=False)

derecha_tipo_counts = df[df['Orientación Política'] == 'Derecha']['Tipo de Think Tank'].value_counts().reset_index()
derecha_tipo_counts.columns = ['Tipo de Think Tank', 'Número de IDs']
derecha_tipo_counts = derecha_tipo_counts.sort_values(by='Número de IDs', ascending=False)

fig_izquierda = px.bar(izquierda_tipo_counts, x='Número de IDs', y='Tipo de Think Tank', 
                       title='Número de IDs por Tipo de Think Tank (Izquierda)', 
                       labels={'Tipo de Think Tank':'Tipo de Think Tank', 'Número de IDs':'Número de IDs'},
                       color='Tipo de Think Tank', orientation='h')

fig_izquierda.show()

fig_derecha = px.bar(derecha_tipo_counts, x='Número de IDs', y='Tipo de Think Tank', 
                     title='Número de IDs por Tipo de Think Tank (Derecha)', 
                     labels={'Tipo de Think Tank':'Tipo de Think Tank', 'Número de IDs':'Número de IDs'},
                     color='Tipo de Think Tank', orientation='h')

fig_derecha.show()




In [17]:
autor_counts = df['Autor'].value_counts().reset_index()
autor_counts.columns = ['Autor', 'Frecuencia']
print(autor_counts)

autor_counts.head(50)

                        Autor  Frecuencia
0               Fundación SOL        1344
1    Universidad de los Andes        1121
2               Bárbara Olave         750
3                Luis Larraín         710
4               Pablo Ortúzar         638
..                        ...         ...
928      Luis Alfonso Herrera           1
929            Martín Verluys           1
930     Gonzalo de la Carrera           1
931         Enrique ter Horst           1
932             POR: PAUTA.CL           1

[933 rows x 2 columns]


Autor  Frecuencia
0              Fundación SOL        1344
1   Universidad de los Andes        1121
2              Bárbara Olave         750
3               Luis Larraín         710
4              Pablo Ortúzar         638
5           Marisol Grisanti         590
6           Claudio Alvarado         491
7    María Ignacia Manríquez         461
8              Lucas Bizarri         439
9              Daniel Mansuy         350
10          Francisco Garcés         347
11             No disponible         340
12         Sylvia Eyzaguirre         311
13               Axel Kaiser         285
14              Luis Cordero         268
15          M. Fernanda Gana         264
16               Jorge Gomez         262
17             Fundación Sol         262
18           Leonidas Montes         244
19              Tomás Flores         234
20            Fernando Claro         218
21            Felipe Larraín         195
22            Florencia Moya         192
23           Francisco Klapp         190
24         David Gallagher\n         188
25        Cristina Del Valle         188
26          Hermann González         178
27                  Chile 21         170
28                Juan Lagos         170
29          María Paz Arzola         166
30            Vittorio Corbo         165
31            Harald Beyer\n         157
32          Natalia González         153
33         Carolina Grünwald         145
34            Pablo Paniagua         139
35         María Jaraquemada         137
36    José Joaquín Brunner\n         133
37            Gerardo Varela         133
38           Roberto Ampuero         125
39              Lucas Sierra         125
40        Evangelina Dardati         122
41            Josefina Araos         117
42             Eduardo Engel         115
43          Lucía Santa Cruz         110
44          Joaquín Castillo         107
45        Joaquín Trujillo\n         107
46     Rodrigo Pérez de Arce         107
47           Nicole Kazazian         106
48            Andrea Repetto         104
49               Pía Mundaca         103

In [18]:
media_counts = df['Medio de Comunicación'].value_counts().reset_index()
media_counts.columns = ['Medio de Comunicación', 'Frecuencia']
print(media_counts)

media_counts.head(50)

                  Medio de Comunicación  Frecuencia
0                           El Mercurio        2712
1                            La Tercera        2470
2                             El Líbero        1126
3                            La Segunda         933
4                     Diario Financiero         578
..                                  ...         ...
704               La Tercera - Página 7           1
705               Publimetro, Página 6.           1
706  Diario Financiero, Páginas 20, 21.           1
707               La Tercera - Página 3           1
708                       PERMISOLOGÍA:           1

[709 rows x 2 columns]


Medio de Comunicación  Frecuencia
0                         El Mercurio        2712
1                          La Tercera        2470
2                           El Líbero        1126
3                          La Segunda         933
4                   Diario Financiero         578
5                        El Mostrador         380
6                               Pulso         287
7          Fundación para el Progreso         273
8                          The Clinic         263
9                            Chile 21         224
10                   Voces La Tercera         167
11                          CNN Chile         135
12                          Teletrece         135
13                            La Hora         124
14                    Espacio Público         107
15                         LA TERCERA          99
16                  Radio Cooperativa          98
17                            Chile B          90
18                        EL MERCURIO          84
19                              Ciper          83
20                          El Dínamo          76
21                           Nodo XXI          75
22                   Revista Qué Pasa          70
23                    El Desconcierto          64
24          El Mercurio de Valparaíso          63
25                               Emol          62
26                        El Pingüino          61
27                            Ex-Ante          61
28                  El Mercurio Legal          52
29                      No disponible          51
30         Radio Universidad de Chile          49
31                              Sabes          48
32                          EL DÍNAMO          45
33                       EL MOSTRADOR          42
34                         Radio Duna          42
35                     Radio Infinita          40
36                  DIARIO FINANCIERO          40
37  El Mercurio - Cuerpo B, Página 2.          37
38                            Capital          36
39              El Mostrador Mercados          34
40                            ESTUDIO          32
41                          Radio ADN          31
42        Diario Universidad de Chile          31
43                         LA SEGUNDA          30
44                 Diarios Regionales          30
45                         El Austral          30
46                      Facebook Live          27
47                 El Mercurio Online          26
48                    Canal CNN Chile          25
49                                IES          24